datadog_cluster_agent/assets/service_checks.json

```
@@ -0,0 +1,17 @@
+[
+    {
+        "agent_version": "6.16.0",
```

Needs to be updated to match the first Agent version this check is expected to be available in, probably 6.24.0 ? (This means 6.24.0 and 7.24.0)

```suggestion
        "agent_version": "6.24.0",
```